### ISSUE TITLE
`Insertable!` fails to compile with slices + array columns

### ISSUE THREAD

**Issue by sgrif** [diesel-rs/diesel#308]
Unsure if it's true for all slices, or just this type specifically. To reproduce:

```rust
struct NewPost<'a> { tags: &'a [&'a str], }
Insertable! { (posts) struct NewPost<'a> { tags: &'a [&'a str], } }
```

**Comment by CryZe** [comment]
I'm not sure if this is the same issue, but diesel_codegen fails to compile on the latest nightly builds (for roughly a week now):

```
/tmp/cargo_393z/.cargo/registry/src/github.com-88ac128001ac3a9a/diesel_codegen-0.6.1/src/insertable.rs:86:46: 86:54 error: no associated item named `empty` found for type `syntax::ptr::P<_>` in the current scope
/tmp/cargo_393z/.cargo/registry/src/github.com-88ac128001ac3a9a/diesel_codegen-0.6.1/src/insertable.rs:86         cx.typaram(span, str_to_ident("DB"), P::empty(), None),
```

**Comment by sgrif** [comment]
@CryZe That was fixed by #312
